### ISSUE TITLE
Blur is not working on flutter web

### ISSUE THREAD

**Issue by DinithHerath** [gskinner/flutter_animate#37]
Hi,

The blur effect is not working on the flutter web giving the following error.
```
══╡ EXCEPTION CAUGHT BY WIDGETS LIBRARY ╞═══════════════════════════════════════════════════════════
The following TypeErrorImpl was thrown building AnimatedBuilder(animation:
AnimationController#697d7(⏮ 0.000; paused)➩Interval(0.6⋯1)➩Tween<Offset>(Offset(0.0, 0.0) →
Offset(4.0, 4.0))➩Offset(0.0, 0.0), dirty, state: _AnimatedState#47719):
Expected a value of type 'JavaScriptObject', but got one of type 'Null'

The relevant error-causing widget was:
  AnimatedBuilder
  AnimatedBuilder:file:///C:/src/flutter/.pub-cache/hosted/pub.dartlang.org/flutter_animate-2.0.1/lib/effects/effect.dart:84:12
```
Can you look on this @gskinner 

Thanks and Regards.

**Comment by gskinner** [comment]
I'll aim to take a look at this soon, but in the interim, have you tested this with the latest version of Flutter and/or master? I'm thinking there was a chance it was fixed by this:
https://github.com/flutter/engine/pull/36575

**Comment by gskinner** [comment]
Fixed in 65749442c98a782998d5019f2fe03bf8109a94bf
Feel free to reopen if you continue to see issues in v2.1 (once it's released).

**Comment by gskinner** [comment]
(fix released in v2.1.0)
